### PR TITLE
Tests to bring 100% coverage

### PR DESCRIPTION
**Pull request by adriguerra**

### CODE REVIEW

File: tests/OptionalTests.cs [hunk 1]
```diff
@@ -73,6 +73,14 @@ where Fail<bool>()
             Assert.That(result, Is.EqualTo(none));
         }
 
+        [Test]
+        public void SelectManySimple() //TODO rename?
+        {
+            var result = Option.Some(42).SelectMany(n => Option.Some(new string((char) n, n)));
```
Comment (atifaziz) [inline tests/OptionalTests.cs]: For consistency with the rest of the tests, this should be also expressed in query syntax, as in:

```c#
var result =
    from n in Option.Some(42)
    from s in Option.Some(new string((char) n, n))
    select s;
```

File: tests/OptionTests.cs [hunk 2]
```diff
@@ -97,6 +97,13 @@ public void ToOptionWithNullableNull()
             Assert.That(result, Is.EqualTo(Option.None<int>()));
         }
 
+        [Test]
+        public void FromTuple()
+        {
+            var result = Option.From((true, 42));
```
Comment (atifaziz) [inline tests/OptionTests.cs]: What about a test for when the first item of the tuple is `false`, as in `Option.From((false, …))`?

File: tests/OptionalTests.cs [hunk 3]
```diff
@@ -73,6 +73,14 @@ where Fail<bool>()
             Assert.That(result, Is.EqualTo(none));
         }
 
+        [Test]
+        public void SelectManySimple() //TODO rename?
```
Comment (atifaziz) [inline tests/OptionalTests.cs]: The name is fine.

**Review by atifaziz — CHANGES_REQUESTED** [review]

File: tests/OptionalTests.cs (hunk 1 above)
Comment (adriguerra) [reply tests/OptionalTests.cs]: Wasn't sure of the difference with the other overload ^^

**Review by atifaziz — APPROVED** [review]

**Pull request merged**